examples/jsm/renderers/webgpu/WebGPUBufferGroup.js

```diff
@@ -0,0 +1,22 @@
+import WebGPUBinding from './WebGPUBinding.js';
+
+class WebGPUBufferGroup extends WebGPUBinding {
```

Comment:
I originally added `WebGPUUniformsGroup` since it represents a _group_ of uniforms. `WebGPUBufferGroup` represents a single `storage-buffer`. How about naming the class `WebGPUStorageBuffer`?

Reply:
Yeah, that name would be clearer.

Similarly, 

```
{
  num: used for compute pass .dispatch(),
  shader: compute shader code strings,
  groups: an array of webgpu binding group
}
```

`groups` should be renamed to, for example `bindings`?

Reply:
Sounds good!

Reply:
Updated, thanks.